docs/conf.py

```diff
@@ -121,10 +122,16 @@
 # -- Options for intersphinx --------------------------------------------------
 
 intersphinx_mapping.update({
-        'ginga': ('https://ginga.readthedocs.io/en/latest/', None)})
+    'ginga': ('https://ginga.readthedocs.io/en/latest/', None)})
 
 # -- Options for linkcheck output ---------------------------------------------
 linkcheck_retry = 5
 linkcheck_ignore = ['https://hsthelp.stsci.edu', 'https://www.ball.com']
 linkcheck_timeout = 180
 linkcheck_anchors = True
+
+# Enable nitpicky mode - which ensures that all references in the docs resolve.
+nitpicky = True
+nitpick_ignore = [
+    ('py:func', 'scipy.ndimage.interpolation.zoom'),
```

Comment:
Why not just change it to `scipy.ndimage.zoom` in the docs?

https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.zoom.html

Reply:
oh good idea, sorry I don't work with `scipy` often so when I do it's difficult to keep track of where everything is